bot/constants.py

```diff
@@ -339,7 +339,7 @@ class _HelpChannels(EnvConfig):
     idle_minutes = 30
     deleted_idle_minutes = 5
     # Roles which are allowed to use the command which makes channels dormant
-    cmd_whitelist: list[int] = [Roles.helpers]
+    cmd_whitelist: list[int] = [Roles.moderators, Roles.admins]
```

Comment:
```suggestion
    cmd_whitelist = Guild.moderation_roles
```
We should use this instead, or better yet, remove this constant entirely and just use `Guild.moderation_roles` in the cog.